src/representor/helper.ts

```diff
@@ -16,34 +16,39 @@ export default class RepresentorHelper {
 
     const defaultTypes: ContentType[] = [
       {
-        mime: 'application/hal+json',
+        mime: 'application/prs.hal-forms+json',
         representor: 'hal',
         q: '1.0',
```

Comment:
Any reason why this mimetype needs to be at the top? Could we just give hal-forms a mimetype of 0.9 like the others?

Reply:
I use Spring HATEOAS for the backend server. It supports multiple media types including HAL and HAL Forms.
If the client indicates to the server that it prefers HAL over HAL Forms (currently the case without my change), the server sends back a simple HAL representation. I think the server behavior is correct and all other servers behave like that.

Reply:
Sounds reasonable!